integration-tests/typings/embrace.d.ts

```diff
@@ -14,4 +14,17 @@ interface EmbraceRequestBody {
   json: EmbracePayload;
 }
 
-export type {EmbraceRequestBody, SessionPayload};
+interface Data {
```

Comment:
will be looking to merge https://github.com/embrace-io/embrace-react-native-sdk/pull/53/files#diff-3c4284e57c0d8ce1dae6e29d723e3535d91d54939e07856eee2b47de6c02d872 shortly which has some changes to the helpers and typings here that we may need to resolve